docs/pages/installation.md

```diff
@@ -248,6 +270,12 @@ Here's what comes in the package.
   - `js/`: Concatenated JavaScript files. Includes minified and unminified files.
     - `plugins/`: Standalone JavaScript plugins.
 
+Once installed with one of the above commands, Foundation has been downloaded via git and installed into the `assets/` directory.
```

Comment:
This is not correct. Each package manager uses its own install mechanism, of those only bower is actually using git, and none of them put Foundation into `assets`.  I think we can probably drop this... lets focus this section on explaining what the pieces are and when you'd use them, as below...